ovos_persona_server/files.py

```
@@ -0,0 +1,352 @@
+import os
+import random
+import string
+import time
+import hashlib
+from contextlib import asynccontextmanager
+from typing import AsyncGenerator, Optional, List
+
+from fastapi import APIRouter, HTTPException, status, UploadFile, File, Form, Query, Depends
+from fastapi import FastAPI
+from fastapi.responses import Response
+from sqlalchemy import select, delete
```

Ruff (0.12.2)

`sqlalchemy.delete` imported but unused

Remove unused import: `sqlalchemy.delete`

(F401)